cards/src/main/resources/cards/custom/Soulseeker/setless/minion_fate_incarnate.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Fate Incarnate",
+  "baseManaCost": 10,
+  "type": "MINION",
+  "heroClass": "DARKBLUE",
+  "baseAttack": 4,
+  "baseHp": 2,
+  "rarity": "EPIC",
+  "race": "SPIRIT",
+  "description": "Opener and Aftermath: Destroy two random enemy minions.",

```

Comment:
this is unplayable level of expensive to have this spell written on a minion, and destroying two enemies on deathrattle is way too strong